CSharp/DevVmPowershell/DevVmPsModules/AddImagesForAllDocumentsModule.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using System.Text;
+using System.Threading.Tasks;
+using Helpers;
+using kCura.Relativity.Client;
+using Relativity.Imaging.Services.Interfaces;
+using Relativity.Services.Search;
+using Relativity.Services.ServiceProxy;
+
+namespace DevVmPsModules
+{
+	[Cmdlet(VerbsCommon.Add, "ImagesForAllDocuments")]
+	public class AddImagesForAllDocumentsModule : BaseModule
+	{
+		[Parameter(
+			Mandatory = true,
+			ValueFromPipelineByPropertyName = true,
+			ValueFromPipeline = true,
+			Position = 0,
+			HelpMessage = "Name of the Relativity Instance")]
+		public string RelativityInstanceName { get; set; }
+
+		[Parameter(
+			Mandatory = true,
+			ValueFromPipelineByPropertyName = true,
+			ValueFromPipeline = true,
+			Position = 1,
+			HelpMessage = "Username of the Relativity Admin")]
+		public string RelativityAdminUserName { get; set; }
+
+		[Parameter(
+			Mandatory = true,
+			ValueFromPipelineByPropertyName = true,
+			ValueFromPipeline = true,
+			Position = 2,
+			HelpMessage = "Password of the Relativity Admin")]
+		public string RelativityAdminPassword { get; set; }
+
+		[Parameter(
+			Mandatory = true,
+			ValueFromPipelineByPropertyName = true,
+			ValueFromPipeline = true,
+			Position = 3,
+			HelpMessage = "Workspace Name that you want to install the application in")]
+		public string WorkspaceName { get; set; }
+
+		protected override void ProcessRecordCode()
+		{
+			//Validate Input arguments
+			ValidateInputArguments();
+
+			IConnectionHelper connectionHelper = new ConnectionHelper(RelativityInstanceName, RelativityAdminUserName, RelativityAdminPassword);
```

Comment:
Please put all the code in one function and call that function from the PS module. It will be much cleaner that way.